README.md

```diff
@@ -40,19 +57,50 @@ For expanded descriptions of the datasets, please see the Pediatric Cancer Data
+Once in the Cavatica workflow page, please click on the "Read All" link to open up the full documentation.
 
 #### Somatic Variant Calling
 
-Small variants are called using multiple tools: Strelka2 for single nucleotide variants (SNVs) and small insertions/deletions (INDELs), GATK Mutect2 for SNVs, multinucleotide variants greater than 1bp in length (MNVs) and INDELs, Lancet for SNVs, MNVs, and INDELs, and VarDict Java which calls SNVs, MNVs, INDELs and more. Larger copy number variants are also called using multiple tools as well: ControlFreeC, CNVkit and GATK CNV. CNVkit calls are adjusted for purity estimations using THeTa2. Manta is also used to determine structural variants (SVs) and INDELs. All calls are made using GRCh38 references and variants are then annotated using gnomAD and for cancer hot spots. Publicly available files are further subjected to “germline masking,” which removes low frequency variants that could be used to identify the sample donor. For more details see the GitHub release at [Kid's First Somatic Variant Workflow](https://github.com/kids-first/kf-somatic-workflow) or to run the pipeline see the [CAVATICA App](https://cavatica.sbgenomics.com/public/apps/cavatica/apps-publisher/kfdrc-somatic-variant-workflow). Once in the Cavatica workflow page, please click on the "Read All" link to open up the full documentation.
+Small variants are called using multiple tools: Strelka2 for single nucleotide variants (SNVs) and small insertions/deletions (INDELs), GATK Mutect2 for SNVs, multinucleotide variants greater than 1bp in length (MNVs) and INDELs, Lancet for SNVs, MNVs, and INDELs, and VarDict Java which calls SNVs, MNVs, INDELs and more. 
+Larger copy number variants are also called using multiple tools as well: ControlFreeC, CNVkit and GATK CNV. 
+CNVkit calls are adjusted for purity estimations using THeTa2. 
+Manta is also used to determine structural variants (SVs) and INDELs. 
+All calls are made using GRCh38 references and variants are then annotated using gnomAD and for cancer hot spots. 
+Publicly available files are further subjected to “germline masking,” which removes low frequency variants that could be used to identify the sample donor. 
+For more details see the GitHub release at [Kid's First Somatic Variant Workflow](https://github.com/kids-first/kf-somatic-workflow) or to run the pipeline see the [CAVATICA App](https://cavatica.sbgenomics.com/public/apps/cavatica/apps-publisher/kfdrc-somatic-variant-workflow). 
+Once in the Cavatica workflow page, please click on the "Read All" link to open up the full documentation. 
+Panel data was not reprocessed and the variant files provided by the submitters were merged with our harmonized data files.
```

Comment:
```suggestion
Panel data was not reprocessed and the variant files provided by the submitters were merged with our harmonized data files.
```

TARGET panel data was reprocessed, and DGD was somewhat - will have to get some details on current DGD processing

Reply:
I had confirmed with Bo that they just took the MAF files for DGD. @zhangb1 is that correct, that you didn't reprocess any DGD files?

Reply:
DGD processed in hg19, so it is a bit complicated. We had rerun their workflow (i believe) in hg38 but then we missed an artifact filtering step - cc @luederm - can you inform on the current DGD release files what was done prior to your coming aboard?